templates/ecs.yaml

```diff
@@ -0,0 +1,317 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Parameters:
+  KeyName:
+    Type: AWS::EC2::KeyPair::KeyName
+    Description: Name of an existing EC2 KeyPair to enable SSH access to the ECS instances.
+  VpcId:
+    Type: AWS::EC2::VPC::Id
+    Description: Select a VPC that allows instances access to the Internet.
+  BastionHostSgId:
+    Type: String
+    Description: The ID of the security group for bastion hosts
+  PublicSubnetId:
+    Type: List<AWS::EC2::Subnet::Id>
+    Description: Public subnet IDs from the VPC for ALB placement. Minimum of two.
+  PrivateSubnetId:
+    Type: List<AWS::EC2::Subnet::Id>
+    Description: Private subnet IDs from the VPC for ECS node placement. Minimum of two.
+  DesiredCapacity:
+    Type: Number
+    Default: '1'
+    Description: Number of instances to launch in your ECS cluster.
+  MaxSize:
+    Type: Number
+    Default: '1'
+    Description: Maximum number of instances that can be launched in your ECS cluster.
+  InstanceType:
+    Description: EC2 instance type
+    Type: String
+    Default: t3a.micro
+    AllowedValues: [t3.micro, t3.small, t3.medium, t3.large, t3a.micro, t3a.small, t3a.medium, t3a.large]
+    ConstraintDescription: Please choose a valid instance type.
+  DockerImageTag:
```

Comment:
would be nice to provide an informative url reference to the available or valid containers.  i'm assuming it's on dockerhub?

Reply:
It is not on Docker hub as we've discussed before, it's built and hosted in a private Gitlab registry.